lms/static/sass/multicourse/_course_about.scss

```diff
@@ -636,4 +614,31 @@
   }
 }
 
+// Sneak-Peek button Styling
```

Comment:
Why not put this in the existing `../stanford/course/_sneakpeek_course_about.scss`?

Reply:
good point